finetune/base_models/huggingface/models.py

```diff
@@ -6,13 +6,100 @@
     TF_BERT_PRETRAINED_MODEL_ARCHIVE_MAP,
     TFBertMainLayer,
     BertTokenizerFast,
-    BertConfig
+    BertConfig,
+
+    XLMRobertaTokenizer,
+    XLMRobertaConfig,
 )
 from transformers.modeling_tf_electra import TFElectraMainLayer
+from transformers.modeling_tf_roberta import TFRobertaMainLayer
+from transformers.tokenization_xlm_roberta import (
+    VOCAB_FILES_NAMES, PRETRAINED_VOCAB_FILES_MAP, PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+)
+from transformers.tokenization_utils import PreTrainedTokenizerFast
+from tokenizers import SentencePieceBPETokenizer
 
 from finetune.util.huggingface_interface import finetune_model_from_huggingface
 
 
+class XLMRobertaTokenizerFast(XLMRobertaTokenizer, PreTrainedTokenizerFast):
+    vocab_files_names = VOCAB_FILES_NAMES
+    pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
+    # pretrained_vocab_files_map['merges_file'] = {"xlm-roberta-base": "/path"}
```

Comment:
I think I want these two lines uncommented, but I'm not completely positive if either configuration yields the exact same tokenizer as the original xlm-roberta (need to confirm with the tests).

The reason why I'm a bit unsure is that the SentencePieceBPETokenizer needs a vocab_file and a merges_file:
https://github.com/huggingface/tokenizers/blob/v0.1.1/bindings/python/tokenizers/implementations/sentencepiece_bpe.py#L21
(logic to use these files is in rust here: https://github.com/huggingface/tokenizers/blob/58e29fd30c4f0b163f145b885732d970a3c387bc/tokenizers/src/models/bpe/model.rs#L249)

The two are typically a *vocab.json and *merges.txt, but in jplu/tf-xlm-roberta-base's case, the vocab_file is a .model file which sentencepiece can take in directly:
https://huggingface.co/transformers/_modules/transformers/tokenization_xlm_roberta.html